05week/spaceTravelToMars.js

```diff
@@ -1,51 +1,87 @@
-'use strict';
+"use strict";
 
-let assert = require('assert');
+let assert = require("assert");
 
 let jobTypes = {
-  pilot: 'MAV',
-  mechanic: 'Repair Ship',
-  commander: 'Main Ship',
-  programmer: 'Any Ship!'
+  pilot: "MAV",
+  mechanic: "Repair Ship",
+  commander: "Main Ship",
+  programmer: "Any Ship!"
 };
 
-// Your code here
+// Your code
+class CrewMember {
+  constructor(name, job, specialSkill) {
+    this.name = name;
+    this.job = job;
+    this.specialSkill = specialSkill;
+  }
+  enterShip(Ship) {
+    Ship.crew.push(this);
+    this.Ship = Ship;
```

Comment:
`this.Ship` should be `this.ship`, since the attribute that is held by the Crewmember class should be `ship` according to the tests